### ISSUE TITLE
Release on F-droid?

### ISSUE THREAD

**Issue by brabalan** [shagr4th/droid48#24]
Is there any hope to release this on F-droid? (https://f-droid.org/)

**Comment by schodet** [comment]
I do not think this is possible as long as the HP48 ROM is bundled with the app. Interface should be added to allow the user to provide its own ROM.
